examples/e14_global_data/src/main.rs

```diff
@@ -0,0 +1,215 @@
+//! In this example, you will be shown various ways of sharing data between events and commands.
+//! And how to use locks correctly to avoid deadlocking the bot.
+
+use std::{sync::{Arc, atomic::{AtomicUsize, Ordering}}, collections::HashMap, env};
+
+use serenity::{
+    async_trait,
+    model::{channel::Message, gateway::Ready},
+    prelude::*,
+
+    framework::standard::{
+        Args, CommandResult, StandardFramework,
+        macros::{command, group, hook},
+    },
+};
+
+use tokio::sync::RwLock;
+
+// A container type is created for inserting into the Client's `data`, which
+// allows for data to be accessible across all events and framework commands, or
+// anywhere else that has a copy of the `data` Arc.
+// These places are usually where either Context or Client is present.
+//
+// Documentation about TypeMap can be found here:
+// https://docs.rs/serenity/0.9.0-rc.2/serenity/prelude/struct.TypeMap.html
+struct CommandCounter;
+
+impl TypeMapKey for CommandCounter {
+    type Value = Arc<RwLock<HashMap<String, u64>>>;
+}
+
+struct MessageCount;
+
+impl TypeMapKey for MessageCount {
+    // While you will be using RwLock or Mutex most of the times you want to modify data,
```

Comment:
"Most of the time" is synonymous with *usually* and is the more correct version here. "Most of the times" would be used in cases where you are specifically talking about occasions when an action/activity happens (or does not).
```suggestion
    // While you will be using RwLock or Mutex most of the time you want to modify data,
```